apps/avifenc.c

```diff
@@ -349,6 +356,52 @@ static avifBool readEntireFile(const char * filename, avifRWData * raw)
     return AVIF_TRUE;
 }
 
+static avifBool avifCodecSpecificOptionsAdd(avifCodecSpecificOptions * options, const char * keyValue)
+{
+    avifBool success = AVIF_FALSE;
+    char ** oldKeys = options->keys;
+    char ** oldValues = options->values;
+    options->keys = malloc((options->count + 1) * sizeof(*options->keys));
+    options->values = malloc((options->count + 1) * sizeof(*options->values));
+    if (!options->keys || !options->values) {
+        free(options->keys);
+        free(options->values);
+        options->keys = oldKeys;
+        options->values = oldValues;
+        return AVIF_FALSE;
+    }
+    if (options->count) {
+        memcpy(options->keys, oldKeys, options->count * sizeof(*options->keys));
+        memcpy(options->values, oldValues, options->count * sizeof(*options->values));
+    }
+
+    const char * value = strchr(keyValue, '=');
+    if (value) {
+        // Remove equals sign.
+        options->values[options->count] = strdup(value);
```

Comment:
strdup isn't a part of C99; it's been added to C23. I guess that's a preexisting condition, though.

Did you want `value + 1` here? In the earlier code the `=` was wiped out with `*value = 0`

Reply:
> strdup isn't a part of C99; it's been added to C23. I guess that's a preexisting condition, though.

Copy-pasted `avifStrdup()` from `avif.c`.

> Did you want `value + 1` here? In the earlier code the `=` was wiped out with `*value = 0`

Done.